### PR TITLE
Update Badge.elm

### PR DESCRIPTION
**Pull request by sufiyanyusuf**
#### :thinking: What?

-Updated typography, paddings
-Fixed contrast issues with danger & success variants

#### :man_shrugging: Why?

- Need to align with the Figma library (The current version looks like from another universe...)

#### :pushpin: Jira Issue

Not tracked.

#### :no_good: Blocked by

Nothing.

#### :clipboard: Pending

- [x] Don't feed the trolls.

### :fire: Extra

...

### CODE REVIEW

**Review by niqt — APPROVED** [review]

**Review by segmentationfaulter — APPROVED** [review]

**Review by miguelcoba — APPROVED** [review]

**Review by PedroHLC — APPROVED** [review]
Morgan Freeman Approves

**Comment by jouderianjr** [comment]
👏 👏 👏 ( an intruder here, but i had to come here to give kudos for this new elm developer )

File: src/UI/Alert.elm [hunk 1]
```diff
@@ -126,12 +130,16 @@ withGenericIcon : Alert msg -> Alert msg
 withGenericIcon (Alert prop opt) =
     Alert prop { opt | genericIcon = True }
 
+isInline : Alert msg -> Alert msg
```
Comment (PedroHLC) [inline src/UI/Alert.elm:133]: Following the with-pattern, you should call this `withInlineStyle`

**Review by PedroHLC — CHANGES_REQUESTED** [review]
Usually, we split the functionality into different PRs to make reviewing easier.

**Pull request merged**